extensions/wikia/Search/templates/WikiaSearch_CrossWiki_index.php

```diff
@@ -10,7 +10,7 @@
 						<?= wfMsg('wikiasearch2-results-for', '<strong>' . $query . '</strong>'); ?>
 					<?php endif; ?>
 					<?php if (isset($hub) && $hub) : ?>
-						<?= wfMsg('wikiasearch2-onhub', $hub) ?>
+						<?= wfMessage( 'wikiasearch2-onhub', Sanitizer::stripAllTags( $hub ) )->text() ?>
```

Comment:
This will still be vulnerable because of the use of `->text()`, since the wikitext will be expanded unescaped after you strip the tags, allowing an attacker to still output raw HTML. Should be `->escaped()` here and below.